src/java.xml/share/classes/com/sun/org/apache/xalan/internal/xsltc/dom/DocumentCache.java

```diff
@@ -168,7 +168,7 @@ public DocumentCache(int size, XSLTCDTMManager dtmManager) throws SAXException {
         _count = 0;
         _current = 0;
         _size  = size;
-        _references = new HashMap<>(_size+2);
+        _references = HashMap.newHashMap(_size);
```

Comment:
Not `_size+2` ?

Reply:
> Not `_size+2` ?

I don't have a idea here why he original use the + 2.
Is there any guy more familiar with this code tell me why?
Thanks!

Reply:
size, not size + 2, same situation as size*2+1 below.